langchain4j-mcp/src/test/java/dev/langchain4j/mcp/client/integration/McpHttpTransportIT.java

```diff
@@ -0,0 +1,100 @@
+package dev.langchain4j.mcp.client.integration;
+
+import static org.assertj.core.api.Assertions.fail;
+
+import dev.langchain4j.mcp.client.DefaultMcpClient;
+import dev.langchain4j.mcp.client.McpClient;
+import dev.langchain4j.mcp.client.transport.McpTransport;
+import dev.langchain4j.mcp.client.transport.http.HttpMcpTransport;
+import java.io.IOException;
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class McpHttpTransportIT extends McpTransportTestBase {
+
+    private static final Logger log = LoggerFactory.getLogger(McpHttpTransportIT.class);
+    private static Process process;
+
+    @BeforeAll
+    public static void setup() throws IOException, InterruptedException, TimeoutException {
+        skipTestsIfJbangNotAvailable();
+        String path = ClassLoader.getSystemResource("tools_mcp_server.java").getFile();
+        String[] command = new String[] {"jbang", "--quiet", "--fresh", "run", path};
+        log.info("Starting the MCP server using command: " + Arrays.toString(command));
+        process = new ProcessBuilder()
```

Comment:
If biggest worry is not requirng jbang then can just add jbang cli as test dep and call the same and it will generate a command line. I'll find that when back at laptop.